### PR TITLE
integration property type chart

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for #766

Before : 
Screenshot 2023-08-11 at 12 09 56 PM
After:
Screenshot 2023-08-11 at 12 10 33 PM

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 24 min

File: django_project/frontend/tests/test_metrics.py [hunk 1]
```diff
@@ -235,3 +239,42 @@ def test_properties_population_category_filter_by_property(self) -> None:
...
+    def test_total_area_per_property_type_filter_by_property(self):
+            """
+            Test total area per property type filtered by property.
+            """
+            id = self.owned_species[0].property_id
+            data = {'property':id}
+            url = self.url
+            response = self.client.get(url, data, **self.auth_headers)
+            self.assertEqual(response.status_code, status.HTTP_200_OK)
+            self.assertEqual(response.data[0]['total_area'], 200)
```
Comment (dimasciput) [inline django_project/frontend/tests/test_metrics.py:280]:
```suggestion
          """
          Test total area per property type filtered by property.
          """
          id = self.owned_species[0].property_id
          data = {'property':id}
          url = self.url
          response = self.client.get(url, data, **self.auth_headers)
          self.assertEqual(response.status_code, status.HTTP_200_OK)
          self.assertEqual(response.data[0]['total_area'], 200)
```

File: django_project/frontend/utils/metrics.py [hunk 2]
```diff
@@ -48,3 +48,23 @@ def calculate_population_categories(queryset: QuerySet) -> Dict[str, int]:
...
+    Calculate the total area per property type
+    for a given queryset of properties.
+    Params:
+        queryset (QuerySet): The queryset of Property objects.
+    Returns:
+        list[dict]: A list of dictionaries, each containing property_type
+                    and total_area keys representing the property type name
+                    and the aggregated total area respectively.
+    """
+    property_ids = [query.id for query in queryset]
```
Comment (dimasciput) [inline django_project/frontend/utils/metrics.py:64]:
```suggestion
    property_ids = queryset.values_list('id', flat=True)
```

**Review by dimasciput — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 30 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 28 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 28 min

**Review by dimasciput — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 26 min

**Pull request merged**